src/app/api/customers/[customerId]/addresses/route.ts

```diff
@@ -36,7 +36,7 @@ export async function GET(
 
     if (!customerAddresses || customerAddresses.length === 0) {
       return responseWrapper(
-        404,
+        200,
```

Comment:
The status code should be 204, but Next Response does not have. This successfully query the address from the customer. However, if the customer does not have any address, it should be No Content instead of Not Found. Not Found is used in the case of error in query when such as the ID cannot be found at all.